pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -1495,11 +1729,33 @@ func (dr *DashboardServiceImpl) searchDashboardsThroughK8sRaw(ctx context.Contex
 			Operator: "in",
 			Values:   query.FolderUIDs,
 		}}
-		if len(request.Options.Fields) == 0 {
-			request.Options.Fields = req
-		} else {
-			request.Options.Fields = append(request.Options.Fields, req...)
-		}
+		request.Options.Fields = append(request.Options.Fields, req...)
+	}
+
+	if query.ProvisionedRepo != "" {
+		req := []*resource.Requirement{{
+			Key:      "repo.name",
+			Operator: "in",
```

Comment:
QQ (non-blocking) – are these defined as constants somewhere that we could use? If not, should we define them as constants?

Reply:
Good idea - k8s has them defined. Updated them :)